pkg/monitortests/network/disruptionpodnetwork/host-network-to-host-network-poller-deployment.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: host-network-to-host-network-disruption-poller
+spec:
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 34%
+      maxSurge: 0
+  # to be overridden by the number of nodes
+  replicas: 1
+  selector:
+    matchLabels:
+      network.openshift.io/disruption-target: host-to-host
+      network.openshift.io/disruption-actor: poller
+  template:
+    metadata:
+      labels:
+        network.openshift.io/disruption-target: host-to-host
+        network.openshift.io/disruption-actor: poller
+    spec:
+      containers:
+        - command:
+            - /usr/bin/openshift-tests
+            - disruption
+            - watch-endpoint-slice
+            - --output-file=/var/log/persistent-logs/disruption-host-to-host.jsonl
+            - --disruption-backend-prefix=host-to-host
+            - --disruption-target-service-name=host-network-service
+            - --stop-configmap=stop-collecting
+            - --my-node-name=$(MY_NODE_NAME)
+          image: docker.io/deads2k/openshift-tests:latest
```

Comment:
Nit: I know these get replaced but wondering if there is value documenting that here like you did in other prs with something like `docker.io/to-be-replaced/openshift-tests:latest`

Reply:
> Nit: I know these get replaced but wondering if there is value documenting that here like you did in other prs with something like `docker.io/to-be-replaced/openshift-tests:latest`

Promised as an IOU